### ISSUE TITLE
Redefining EXPORT'ed symbols in user programs.

### ISSUE THREAD

**Issue by Matthew-McRaven** [StanWarford/pep10suite#10]
Starting the Pep10 assembler we allow users to EXPORT the values of symbols from the operating system into the user program. This feature is necessary to enable features such system call macros, and better handling of memory-mapped IO devices.

When a user program is assembled, all symbols marked with EXPORT in the operating system are copied into the user symbol table. If a user attempts to redefine any EXPORT'ed symbol inside the user program, they will be notified that the symbol is multiply defined, and user program assembly will error.

The inability to redefine EXPORT'ed symbols in a user program seems a reasonable restriction to me, but @StanWarford I wanted your thoughts.

For example, in the following pair, the OS will assemble successfully, and the user program will fail with the error ";ERROR: Symbol "sym" was previously defined."

OS
```
     .BURN   0xFFFF
sym: .EQUATE 20
     .EXPORT sym
     .END
```

User Program
```
sym: .EQUATE 10
     .END
```

**Comment by StanWarford** [comment]
Yes. In fact, I assumed this was already the case.

**Comment by Matthew-McRaven** [comment]
This is the current behavior.

I wanted to verify this behavior, as it might not be obvious to users why
they are getting “multiply defined” errors when overriding an OS symbol.

In the example I provided, a user only defined “sym” once, but received a
“multiply defined” error because of a previous definition in the operating
system. Inexperienced users may struggle to diagnose the root cause of this
issue (i.e. a clash with the operating system).

One way to alleviate this would be to add a more specific error message
when redefining a symbol from the operating system like “Error: Attempted
to redefine “sym”, which was previously declared in the Operating System”

**Comment by StanWarford** [comment]
Good idea.

**Comment by Matthew-McRaven** [comment]
The error message is as follows, where %1 is the offending symbol:
`;ERROR: Attempted to redefine symbol\"%1\", which was exported from the operating system.`

Does this seem sufficient?

**Comment by StanWarford** [comment]
Minor edit
`;ERROR: Attempted to redefine symbol \"%1\" exported from the operating system.`

**Comment by Matthew-McRaven** [comment]
Fixed with a8b5dda.
